src/schemas/fairdata-ida.ui.js

```diff
@@ -9,7 +9,7 @@ export default {
 		{ label: 'Extra', uri: null },
 	],
 	//'': { 'tab': 'description' },
-	'': { 'tab': 'extra', 'order': ["title", "description", "language"] },
+	'': { 'tab': 'extra', 'order': ["title", "description", "issued", "language"] },
```

Comment:
why this change, the /properties/issued seems to be marked for the `description` tab which was a change also from the previous value. see other comment.

Reply:
This makes "issued" field appear after description and before language.

It's a bit misleading at first, but the tab and order are unrelated here. The line does two separate things for the dataset root:
- 'tab': 'extra' makes 'extra' the default tab. Items that don't have tab set will be in the 'extra' tab, which is hidden. 
- 'order': [...] defines the order in which the items appear. First, we show items in the array order and then the remaining items in their default order. This is shared across all tabs.

Reply:
Thanks for explaining this part 👍